module/Activity/view/activity/activity/create.phtml

```diff
@@ -5,9 +5,10 @@ $this->headLink()
 $this->headScript()
     ->prependFile($this->basePath() . '/js/activity-create.js')
     ->prependFile($this->basePath() . '/js/jquery.datetimepicker.full.min.js');
-$form->prepare();
+if (!$success) {$form->prepare();}
 ?>
 <div class="container">
+    <?php if (!$success): ?>
```

Comment:
In that case I think this is fine for now, since the if-else block will leave when we add the redirect.